sdk/core/Azure.Core/src/DiagnosticsOptions.cs

```diff
@@ -81,10 +83,15 @@ internal DiagnosticsOptions(DiagnosticsOptions? diagnosticsOptions)
         public bool IsLoggingEnabled { get; set; } = true;
 
         /// <summary>
-        /// Gets or sets value indicating whether distributed tracing spans are going to be created for this clients methods calls and HTTP calls.
+        /// Gets or sets value indicating whether distributed tracing spans are going to be created for the clients methods calls and HTTP calls.
         /// </summary>
         public bool IsDistributedTracingEnabled { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets value indicating whether nested distributed tracing spans are going to be created for client methods when called by other instrumented client methods.
+        /// </summary>
+        public bool IsNestingClientSpansEnabled  { get; set; }
```

Comment:
discussed naming with @annelo-msft and @ShivangiReja. The suggestion is to have a flag that's `false` by default and we'll override it in new DPG clients to `true` to make it less confusing.

then the name should be something along the lines of `IsNestedSpanSuppressionEnabled`

Reply:
In that case, why not go back to the initial name of SupressNestedSpans or the like? I find IsNestedSpanSuppressionEnabled even more confusing than it is currently.

Reply:
@JoshLove-msft thank you for the suggestion, I was trying to follow the existing pattern of `Is*Enabled`, but `SupressNestedSpans` is much better.